VideoLocker/src/main/java/org/edx/mobile/third_party/iconify/IconDrawable.java

```diff
@@ -1,47 +1,40 @@
 package org.edx.mobile.third_party.iconify;
 
-/**
- * Copyright 2013 Joan Zapata
```

Comment:
Alternately, since this is so heavily modified - and brings in edx specific code should we integrate properly into the repo?